src/types/webhooks/create_webhook.ts

```diff
@@ -0,0 +1,11 @@
+import { SnakeCaseProps } from "../util.ts";
+
+export interface CreateWebhook {
+  /** Name of the webhook (1-80 characters) */
+  name: string;
+  /** Image for the default webhook avatar */
+  avatar: string | null;
+}
+
+/** https://discord.com/developers/docs/resources/webhook#create-webhook-json-params */
+export type DiscordCreateWebhook = SnakeCaseProps<CreateWebhook>;
```

Comment:
You don't need to use `SnakeCaseProps` on the interface since the type does not have properties that need to be converted to snake case, this is redundant.